vj4/model/adaptor/training.py

```diff
@@ -8,16 +8,17 @@
 
 
 @argmethod.wrap
-async def add(domain_id: str, title: str, content: str, owner_uid: int, dag=[]):
+async def add(domain_id: str, title: str, intro: str, content: str, owner_uid: int, dag=[]):
```

Comment:
Sorry, it should be here.

Inserting `intro` between `title` and `content` will break the pattern of parameters.
See:
* https://github.com/vijos/vj4/blob/master/vj4/model/adaptor/discussion.py#L133
* https://github.com/vijos/vj4/blob/master/vj4/model/adaptor/problem.py#L37

If introduction is more important, let `content` store introduction and another field store description.